dir.traversal.targets

```diff
@@ -86,6 +86,32 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
+  <Target Name="ProducesPackageId"
+          Returns="@(PackageIds)" 
```

Comment:
>Also, I think that a full build would be a terrible pre-req for the produces api.

Do you know in what context someone will be calling Produces without doing an actual build? I ask mainly because the versions you get will be different depending one when you run the Produces and we actually get a Build.